stackdriver_trace.c

```diff
@@ -599,7 +604,8 @@ PHP_FUNCTION(stackdriver_trace_list)
     int i = 0;
     stackdriver_trace_span_t *trace_span;
     int num_spans = STACKDRIVER_TRACE_G(spans)->nNumUsed;
-    zval labels[num_spans], spans[num_spans];
+    zval *labels = emalloc(num_spans * sizeof(zval));
```

Comment:
I'm not familiar with the PHP internal memory management, but is it safe to do:
emalloc -> array_init -> (and array_init in stackdriver_labels_to_zval_array) -> efree
?